### DISCUSSION TITLE
Google PubSub

### DISCUSSION THREAD

**Discussion by nathan-malinowski** [nextgenhealthcare/connect#5813] [Q&A]
I am looking to get HL7 messages pulled HCAPI via a Google PubSub feed and was looking to create a plugin source connector to do so, but wanted to see if there was maybe an easier way to achieve this goal. I am looking into a Javascript Reader, but was wondering if setting up a JMS Listener would work and if anyone has tried to do something like this yet?

**Comment by jonbartels** [comment]
Can you show the code you implemented? 

You posted in slack but having it here makes it more persistent and searchable
